analytics.go

```diff
@@ -113,3 +166,28 @@ func (c *Client) GetAggregatedTeamData(ctx context.Context, analytics AnalyticsR
 
 	return analyticsResponse, nil
 }
+
+// Get raw data multiple incidents gets the incidents and its metrics for the requested data.
+func (c *Client) GetMultipleRawIncidents(o IncidentAnalyticsRequest) (AnalyticsIncidentResponse, error) {
+	return c.GetMultipleRawIncidentsWithContext(context.Background(), o)
+}
+
+// Gets raw data multiple incidents with context gets the incidents and its metrics for the requested data.
```

Comment:
This GoDoc comment is not formatted correctly. It should start with the method name, and then explain what it is / does.

In addition, could you add a separate newline separated section below indicating it's also using an early access API, and could be subject to change? We probably can't explicitly guarantee API compatibility until the REST API itself is stable.

So, including other suggested changes, it would be something like:

```Go
// GetRawIncidentsAnalytics gets the raw analytics for multiple incidents based on
// the filter provided within the AnalyticsRawIncidentsRequest.
//
// This method currently uses an Early Access REST API endpoint, and as a result
// may be subject to change until the API endpoint becomes stable.
```